osquery/core/process.h

```diff
@@ -37,7 +37,7 @@ using PlatformPidType = pid_t;
 #endif
 
 /// Constant for an invalid process
-const PlatformPidType kInvalidPid = (PlatformPidType) - 1;
+const PlatformPidType kInvalidPid = (PlatformPidType)-1;
```

Comment:
This is an artifact of running two versions of `clang-format`. If someone runs a 3.4 or 3.6 and then a 3.8 comes in, it may choose to improve the code. Then someone with a 3.4 comes back, makes edits, and see-saws the formatting back.

I have a plan to converge on a version, but we may also want to investigate backwards and forwards compatible formatting configuration.